toolkit/tools/internal/processes/processes.go

```diff
@@ -79,6 +106,28 @@ func GetProcessesUsingPath(path string) ([]ProcessRecord, error) {
 	return records, nil
 }
 
+func getLsofVersion() (int, int, error) {
+	_, stderr, err := shell.NewExecBuilder("lsof", "-v").
+		LogLevel(logrus.TraceLevel, logrus.TraceLevel).
+		ExecuteCaptureOuput()
```

Comment:
I guess this function initially had a wrong name typo - `ExecuteCaptureOu**t**put()`, but could you help to fix it?

Reply:
I've opened a bug.